multimodal/tests/unittests/test_object_detection.py

```diff
@@ -35,7 +35,7 @@ def test_mmdet_object_detection_inference(checkpoint_name):
         pipeline="object_detection",
     )
 
-    pred = predictor.predict({"image": [mmdet_image_name] * 10})  # test batch inference
+    pred = predictor.predict({"image": [mmdet_image_name] * 10}, realtime=False)  # test batch inference
```

Comment:
Do we want to add a test coverage for automatic inference or `realtime=True`?

Reply:
Currently, `realtime=True` would cause at least two different errors for object detection. I will fix it in the next PR.